test/spec/ViewerSpec.js

```diff
@@ -742,8 +742,8 @@ describe('Viewer', function() {
           viewer.importXML(multipleXML);
 
           // then
-          viewer.on('import.done', function(event) {
-            done();
+          viewer.on('import.done', function(context) {
```

Comment:
It is actually an event, not some context and I'd keep the _event_ identifier because of that.